utils/s3put.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+if [ -z "$1" ]; then
+    echo "Usage: $(basename $0) model-directory"
+    exit 1
+fi
+
+MODEL=$1
+
+s3cmd put --recursive --acl-public $MODEL s3://fb-glow-assets/models/
```

Comment:
I assume this requires having AWS_ACCESS_KEY, etc to be set? Can you put comments in the file about those?

Reply:
by comments I mean: Make sure to set those Keys blah blah blah.